internal/config/config.go

```diff
@@ -107,6 +107,13 @@ var config struct {
 		Heartbeat   time.Duration `env:"HEARTBEAT" env-default:"30s" env-description:"heartbeat interval (time interval syntax)"`
 		MaxBeats    int           `env:"MAX_BEATS" env-default:"10" env-description:"maximum amount of heartbeats allowed"`
 	} `env-prefix:"WORKER_"`
+	Unleash struct {
+		Enabled     bool   `env:"ENABLED" env-default:"false" env-description:"unleash service (feature flags)"`
+		Environment string `env:"ENVIRONMENT" env-default:"" env-description:"unleash environment"`
+		Prefix      string `env:"PREFIX" env-default:"app.provisioning" env-description:"unleash flag prefix"`
```

Comment:
Also environment is configurable option now, this will get used if we will have a single instance across environments, which is not the case I think. But I added it, just do not set it and an empty string is the default value that should not hurt anything in the client.